docs/spec/interfaces.md

```diff
@@ -0,0 +1,93 @@
+# Interface Contracts
+
+<!-- This image is sourced from https://drive.google.com/open?id=1Mgn0Oz85M9W1Pwfnw3u0OhZmIIxSxcM3lZUEZyPCoB0 -->
+![Interface Contracts Overview](images/interface-contracts-overview.svg)
+
+---
+
+## Subscribable
+
+A **Subscribable** resource will emit events that a _Subscription_ can direct
+to a _Targetable_ or _Sinkable_ resource.
```

Comment:
s/to a ... resource/to their configured destinations/

(There's no need to reference the other interfaces when defining Subscribable.

Reply:
thanks